core/build.sbt

```diff
@@ -359,6 +359,10 @@ lazy val operator =
 
       publishArtifact in (Compile, packageDoc) := false,
       publishArtifact in (Compile, packageSrc) := false,
+      // skuber version 2.4.0 depends on akka-http 10.1.9 : hence overriding
+      // with akka-http 10.1.11 to use akka 2.6.3
+      // remove this override once skuber is updated
+      dependencyOverrides += "com.typesafe.akka" %% "akka-http" % Version.AkkaHttp,
```

Comment:
I would just add a simple dependency on the pre-defined AkkaHttp dependency in the main dependencies list of the project. It will have the same effect and will be slightly easier to maintain.

Reply:
We already do the same for the `AkkaStream` and `AkkaSlf4j` dependencies, probably for the exact same reason.

Reply:
@agemooij  do you mean to use `force()` ?

```
val AkkaHttp  = "com.typesafe.akka"   %% "akka-http"   % Version.AkkaHttp force()
```

Reply:
No, simpler than that. Just add a normal, run of the mill dependency to the right version. As long as the version we want is the latest version, the normal deendency evition mechanism will use that instead of any outdated transitives.

So just add it like this:

```
    .settings(
      libraryDependencies ++= Vector(
        AkkaSlf4j,
        AkkaStream,
        AkkaHttp,     // <-----
        Ficus,
        .....
```

Reply:
Also let's add a comment somewhere to take a note for this behavior.

Reply:
ok .. if it works the same way as `dependencyOverrides` it should be fine ..

Reply:
But I chose to stick with `dependencyOverrides` as sbt doc recommends it to be the preferred way